apps/condo/lang/en/en.json

```diff
@@ -1577,5 +1577,65 @@
   "billing.category.internet.name": "Internet",
   "billing.category.radio.name": "Radio",
   "billing.category.intercom.name": "Intercom",
-  "billing.category.telecom.name": "Telecom"
+  "billing.category.telecom.name": "Telecom",
+  "banking.integration.SBBOL.name": "SBBOL",
+  "banking.integration.1CClientBankExchange.name": "1C",
+  "banking.category.maintenance_and_current_repair_of_common_property_in_an_apartment_building.name": "Maintenance and current repair of common property in an apartment building",
+  "banking.category.apartment_building_management.name": "Apartment building management",
+  "banking.category.overhaul.name": "Overhaul",
+  "banking.category.drainage.name": "Drainage",
+  "banking.category.water_supply.name": "Water supply",
+  "banking.category.garbage_collection_and_disposal.name": "Garbage collection and disposal",
+  "banking.category.gas.name": "Gas",
+  "banking.category.hot_water_supply.name": "Hot water supply",
+  "banking.category.additional_services.name": "Additional services",
+  "banking.category.thermal_energy.name": "Thermal energy",
```

Comment:
If third token of intl key is a constant - you need to use upper snake case for that

Reply:
For example, 
```suggestion
  "banking.category.THERMAL_ENERGY.name": "Thermal energy",
```